docker-bake.hcl

```diff
@@ -122,12 +122,38 @@ target "binaries-cross" {
   ]
 }
 
-target "release" {
+target "pkg" {
+  inherits = ["binaries"]
+  args = {
+    PKG = "apk,deb,rpm"
```

Comment:
Maybe apk is not necessary as buildx is already available for alpine: https://pkgs.alpinelinux.org/packages?name=*buildx*&branch=v3.14

Reply:
fwiw, here's the `APKBUILD` that we use.  we regularly rebuild for new `go` versions, CVEs, etc.

https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/docker-cli-buildx/APKBUILD

Reply:
@tomalok sgtm thanks for your feedback